ckks/linear_transform.go

```diff
@@ -74,7 +73,7 @@ type LinearTransform struct {
 	LogSlots int                 // Log of the number of slots of the plaintext (needed to compute the appropriate rotation keys)
 	N1       int                 // N1 is the number of inner loops of the baby-step giant-step algorithm used in the evaluation (if N1 == 0, BSGS is not used).
 	Level    int                 // Level is the level at which the matrix is encoded (can be circuit dependent)
-	Scale    float64             // Scale is the scale at which the matrix is encoded (can be circuit dependent)
+	Scale    *rlwe.Scale         // Scale is the scale at which the matrix is encoded (can be circuit dependent)
```

Comment:
Switch to non-pointer type ? (depends on if we want `Scale` field to possibly be `nil`)

Reply:
Done